app-web/src/templates/StandAlone_GitHubRaw.js

```diff
@@ -20,30 +20,71 @@ import React from 'react';
 import { graphql } from 'gatsby';
 import Layout from '../hoc/Layout';
 import { Main } from '../components/Page';
-import CardHeader from '../components/Cards/Card/CardHeader';
 import rehypeReact from 'rehype-react';
 import styled from '@emotion/styled';
+import Pill from '../components/UI/Pill';
+import { withPadding } from '../components/GithubTemplate/common';
+import slugify from 'slugify';
 
-const PageDiv = styled.div`
-  padding-top: 20px;
-`;
 const ContentDiv = styled.div`
   padding-top: 10px;
 `;
+const PillDiv = styled.div`
+  padding: 5px 0px;
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  :p {
+    margin: auto;
+  }
+`;
+
+const IconPill = styled(Pill)`
+  margin-top: 0px;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+const Header = styled.header`
+  background-color: #f1f1f1;
+  border-bottom: 1px solid #ccc;
+  width: 100%;
+  ${withPadding}
+`;
 
 export const StandAloneGitHubRawResource = ({ data: { githubRaw } }) => {
-  const isExternal = false;
   const renderAst = new rehypeReact({
     createElement: React.createElement,
   }).Compiler;
 
+  let topics = githubRaw.fields.topics;
+  let topicPills = [];
+  if (topics) {
+    topicPills = topics.map(topic => {
+      return (
+        <a href={`/${topic.fields.slug}/${githubRaw.fields.slug}`} key={topic.id}>
```

Comment:
use the `Link` Component for non web uris

Reply:
good call, consider it done